debug-ui/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "start:feed": "tsx scripts/start-fill-feed.ts",
+    "run:rando-bot": "tsx scripts/run-rando-bot.ts",
+    "run:pk-solflare-to-phantom": "tsx scripts/pk-solflare-to-phantom.ts",
+    "lint": "tsc --noEmit && next lint",
+    "lint:fix": "tsc --noEmit && next lint --fix"
+  },
+  "dependencies": {
+    "@cks-systems/manifest-sdk": "file:../dist/cjs",
```

Comment:
Thoughts on if the typescript client is complete and respectable enough to publish and we can just reference a fixed version?

Reply:
Everything I used so far seemed fine to me. No complaints other than the lack of UI instantiation method which now exists. No complaints from me.